doc/en/example/parametrize.rst

```diff
@@ -411,8 +411,6 @@ argument sets to use for each test function.  Let's run it:
         def test_equals(self, a, b):
     >       assert a == b
     E       assert 1 == 2
-    E         -1
```

Comment:
Those changes are actually correct, they were a recent fix by how "verbose" was handled internally by some plugins (`-q` was being treated as `-v` in some cases.